effi-ui-clean-inject.sh

```
@@ -0,0 +1,279 @@
+}
+
+# Print functions
+print_success() { echo -e "${GREEN}✓ $1${NC}"; }
+print_error() { echo -e "${RED}✗ $1${NC}"; }
+print_warning() { echo -e "${YELLOW}⚠ $1${NC}"; }
+print_info() { echo -e "${BLUE}ℹ $1${NC}"; }
+
+# Create backup
+backup_file() {
+    local file="$1"
+    local filename=$(basename "$file")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)